tests/robotinterface/RobotInterfaceTest.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+<robot name="robotinterface_additional_controlboard_nws" prefix="robotinterface_additional_controlboard_nws">
+  <devices>
+    <device name="additional_controlboard_nws" type="controlboardwrapper2">
```

Comment:
Where do you think this explanation should go? There is already a comment later in this file, probably it could make more sense to expand that and move it here?

Reply:
I was looking for a simple comment that complements the comment later in the file. 

> There is already a comment later in this file, probably it could make more sense to expand that and move it here?

This would do the job.

Reply:
Comment expanded and updated in https://github.com/robotology/gazebo-yarp-plugins/pull/532/commits/6d68489719d72a66c279996b1bd946d3fa4861d2 .